pz_be_services/routers/v1/ai_router.py

```
@@ -0,0 +1,60 @@
+            messages=[
+                {
+                    "role": "system",
+                    "content": "You are a helpful assistant.Answer in poetry",
+                },
+                {"role": "user", "content": input},
+            ],
+            max_completion_tokens=1310,
+        )
+        return {"response": response.choices[0].message.content}
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)